utils.go

```diff
@@ -146,3 +166,35 @@ func (s *Storage) getFileName(path string) string {
 		return path
 	}
 }
+
+func initCache(nc int64, mc int64, bi int64) (cache *Cache, err error) {
+	config := &Config{
+		NumCounters: nc,
+		MaxCost:     mc,
+		BufferItems: bi,
+	}
+
+	cache, err = NewCache(config)
+
+	if err != nil {
+		return nil, err
+	}
+	return cache, nil
+}
+
+func (s *Storage) setCache(path string, fileId string) (success bool) {
+	success = s.cache.SetWithTTL(path, fileId, cost, expireTime)
+	if success {
+		return true
+	} else {
+		return false
+	}
+}
```

Comment:
```suggestion
func (s *Storage) setCache(path string, fileId string) {
	s.cache.SetWithTTL(path, fileId, cost, expireTime)
}
```

It looks we don't care about the result about set.